src/lib/utils/skin.ts

```diff
@@ -175,19 +175,25 @@ export function getFadeCalculatorAndSupportedWeapon(
     for (const [fadeType, calculator] of Object.entries(FADE_TYPE_TO_CALCULATOR)) {
         for (const supportedWeapon of calculator.getSupportedWeapons()) {
             if (asset.market_hash_name.includes(`${supportedWeapon} | ${fadeType}`)) {
-                return [calculator, supportedWeapon.toString()];
+                return [calculator, supportedWeapon.toString(), fadeType.replace(' ', '-').toLowerCase()];
```

Comment:
I think we're starting to overload the term "Fade Type" -- we have both the paint name mapping above and this secondary form of "Fade Type" that is used for the class.

I'd prefer to make this a more explicit mapping and call it a different term (perhaps `class_name`). We're at a point where I think returning an object instead of an array is the way to go, especially with two back to back strings.